utils/src/main/kotlin/Constants.kt

```diff
@@ -35,6 +35,11 @@ const val ORT_FULL_NAME = "OSS Review Toolkit"
  */
 const val ORT_CONFIG_DIR_ENV_NAME = "ORT_CONFIG_DIR"
 
+/**
+ * The name of the environment variable to customize the ORT tools directory.
+ */
+const val ORT_TOOLS_DIR_ENV_NAME = "ORT_TOOLS_DIR"
```

Comment:
Just wondering, should we call this more specifically "ORT_BOOTSTRAP_DIR"?

Reply:
I prefer "tools", I think it makes it more clear that this is about external tools. "ORT_BOOTSTRAP_DIR" could also mean that ORT itself is bootstrapped.